deploy.sh

```diff
@@ -90,18 +102,18 @@ case "$DEPLOYMENT_TYPE" in
     dev)
         API_PORT=$(expr $API_PORT_BASE + 100)
         UI_PORT=$(expr $UI_PORT_BASE + 100)
-        PROJECT_NAME="$LOGIN_USER-dev"
+        STACK_NAME="$APP_NAME-$LOGIN_USER-dev"
```

Comment:
"-dev" is not needed: the user name alone is enough to indicate this is a development deployment (barring having users named "prod" or "staging"), please change this to be consistent with story-indexer